arangod/Cluster/TraverserEngine.cpp

```diff
@@ -189,9 +189,10 @@ void BaseEngine::getVertexData(VPackSlice vertex, VPackBuilder& builder,
       return;
     }
     std::string_view vertex = id.substr(pos + 1);
-    for (std::string const& shard : shards->second) {
+    for (auto const& shard : shards->second) {
       Result res = _trx->documentFastPathLocal(
-          shard, vertex, [&](LocalDocumentId const&, VPackSlice doc) {
+          std::string{shard}, vertex,
```

Comment:
can we avoid this conversion to string here? I think documentFastPathLocal now accepts a string_view, so we can remove the conversion now.

Reply:
Good catch! Added in: 9734b57526c551cc0fe071bbc0d89eddae8cf6e6